definitions/tools/optitype_dna.wdl

```diff
@@ -7,19 +7,22 @@ task optitypeDna {
     File cram_crai
     File reference
     File reference_fai
+    Int threads = 8
+    Int mem = 50
   }
 
   Int space_needed_gb = 10 + round(5*size([cram, cram_crai, reference, reference_fai], "GB"))
   runtime {
-    memory: "64GB"
-    docker: "mgibio/immuno_tools-cwl:1.0.1"
+    memory: "~{mem}GB"
+    cpu: threads 
+    docker: "laljorani20/immuno-tools:latest"
```

Comment:
I'd encourage using particular tagged versions instead of relying on a `latest` version that might change.

Reply:
If you'd like to make a PR against the genome repo for the docker image I can publish an updated version to `mgibio`.

Reply:
PR to the docker image should be linked at the end of this PR. Appreciate your help!

Reply:
A new tag has been pushed for version [mgibio/immuno_tools-cwl:1.0.2](https://hub.docker.com/layers/immuno_tools-cwl/mgibio/immuno_tools-cwl/1.0.2/images/sha256-c7cea4eb50576c67bccbceee586215e1255b06ca01d46d31b1421a4579d8d4dc).

(As a side note, you may need to rebase this branch against the current master to get the "verify" check to run on it.)